tasklib/backends.py

```diff
@@ -245,7 +245,7 @@ def filter_class(self):
 
     @property
     def config(self):
-        # First, check if memoized information is available
+        # First, check if memorized information is available
```

Comment:
This isn't a typo 😄 it refers to [memoization](https://en.wikipedia.org/wiki/Memoization) (I know, weird word!)

Reply:
Ouups, sorry, I didn't know the word nor my corrector.

Restored on cf4d52a